zschema/__main__.py

```diff
@@ -27,26 +28,15 @@ def main():
         print json.dumps(record.to_bigquery())
     elif command == "elasticsearch":
         print json.dumps(record.to_es(recname))
+    elif command == "es-annotated":
+        print json.dumps(record.to_es(recname, annotated=True))
```

Comment:
What's `es-annotated` mean? Why would we output annotated data to elasticsearch?